src/mocompiler.js

```diff
@@ -183,9 +222,9 @@ Compiler.prototype._calculateSize = function (list) {
 /**
  * Generates the binary MO object from the translation list
  *
- * @param {Array} list translation list
- * @param {Object} size Byte size information
- * @return {Buffer} Compiled MO object
+ * @param {TranslationBuffers[]} list translation list
+ *  @param {Size} size Byte size information
+ *  @return {Buffer} Compiled MO object
```

Comment:
We should remove these spaces... also would be nice to add a JSDoc eslint rules.

Reply:
yep you are right, it seems that jsdocs aren't formatted at all. 

What do you suggest to fix that issue?

Reply:
@erikyo I've got a branch that is almost done, in which I manually format them. I looked into using `eslint-plugin-jsdoc` though I was having some trouble because `eslint-config-standard` hasn't be updated (by it's maintainers) and I would need to figure out what version of `eslint-plugin-jsdoc` should be added as a dev dependency. If there is support to add the JSDoc plugin, I can figure it out, but didn't want to commit much time to it, if there wasn't support.